grype/presenter/sarif/test-fixtures/snapshot/TestSarifPresenterImage.golden

```diff
@@ -69,8 +69,8 @@
               },
               "logicalLocations": [
                 {
-                  "name": "etc/pkg-1",
-                  "fullyQualifiedName": "user-input@sha256:asdf:/etc/pkg-1"
+                  "name": "/foo/bar/somefile-1.txt",
+                  "fullyQualifiedName": "user-input@:/somefile-1.txt"
```

Comment:
Not having location specific generation in the mock is a small limitation - happy to switch on the scheme to generate correct location data depending on IMG vs Dir scheme